app/components/system_header/index.tsx

```diff
@@ -39,18 +40,24 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
         header: {
             flex: 1,
             flexDirection: 'row',
+            marginTop: 10,
+        },
+        timeContainer: {
+            flex: 1,
+            marginTop: 5,
+            flexDirection: 'row',
+            alignItems: 'center',
```

Comment:
I believe alignItems might not be needed.  Could you confirm?

Reply:
checked, its not needed